src/cloudConsoleLauncher.ts

```diff
@@ -251,16 +265,19 @@ export async function delay(ms: number) {
 	return new Promise<void>(resolve => setTimeout(resolve, ms));
 }
 
-export async function runInTerminal(accessToken: string, consoleUri: string) {
-	process.stdin.setRawMode!(true);
-	process.stdin.resume();
+export async function runInTerminal(accessToken: string, consoleUri: string, tempFile: string) {
+	if (tempFile) {
```

Comment:
hmm... i wouldn't mind more comments here. do we really have to check if tempFile is not null here?
why should anyone call this function with tempFile == null

Reply:
tempfile= null is scenario where caller doesn't care of terminal status, they just trigger connect to terminal, then return

Reply:
ah, ok, i understand now